src/App.js

```diff
@@ -27,7 +26,14 @@ class App extends React.Component {
         {/* eslint-disable-next-line */}
         <h1>Carousel with {images.length} images</h1>
 
-        <Carousel />
+        <Carousel
+          images={this.state.images}
```

Comment:
```suggestion
          images={images}
```
Ты же сделала деструктуризацию